pre-commit

```diff
@@ -130,12 +130,16 @@ for changedfile in $files_to_check; do
     #puppet manifest styleguide compliance
     if [[ "$CHECK_PUPPET_LINT" != "disabled" ]] ; then
         if type puppet-lint >/dev/null 2>&1; then
-            if [[ $(echo "$changedfile" | grep -q '\.*\.pp$' ; echo $?) -eq 0 ]]; then
+            if echo "$changedfile" | grep -iq '\.pp$'; then
+              if [[ 0"$failures" -eq 0 ]]; then
```

Comment:
It's a deliberate leading zero, to catch the case where $failures is empty - otherwise you get [[ "" -eq 0 ]], which throws an error.

Alternative would be to initialize to 0 before the loop - either is fine, this is just my defensive programming habits kicking in.  I've been burned by empty variables too many times in the past.